### DISCUSSION TITLE
After getting an access token, how can you use it with the msgraph library?

### DISCUSSION THREAD

**Discussion by neilkumar** [rayluo/identity#32] [Q&A] [answered]
I'm trying to use https://github.com/microsoftgraph/msgraph-sdk-python to make calls to the Graph API. How do I use the access_token I receive after the user logins/consents?

**Comment by rayluo** [comment, marked as answer]
Looks like the [Microsoft Graph SDK is tailored to accept a credential defined in Azure Identity](https://github.com/microsoftgraph/msgraph-sdk-python?tab=readme-ov-file#3-make-requests-against-the-service), rather than a raw access token. It is probably feasible to implement a thin wrapper credential to feed the access token into Microsoft Graph SDK. UPDATE: [It is feasible](https://github.com/microsoftgraph/msgraph-sdk-python/issues/501).

**Comment by neilkumar** [comment]
Ok, I got it to work, this was helpful https://github.com/microsoftgraph/msgraph-sdk-python/issues/501

To be honest, somewhat surprised this is not a common use case. 🤷 

Thank you for your help!
